applications/ConstitutiveLawsApplication/custom_constitutive/small_strain_isotropic_damage_3d.cpp

```diff
@@ -72,6 +73,10 @@ bool SmallStrainIsotropicDamage3D::Has(const Variable<Vector>& rThisVariable)
 {
     if(rThisVariable == INTERNAL_VARIABLES){
         return true;
+        
+    } else if(rThisVariable == STRAIN){
+        // explicitly returning "false", so the element calls CalculateValue(...)
+        return false;
```

Comment:
i would say that this one is not needed. if it is not find, it will return false